tls/test_tls_handshake.py

```diff
@@ -97,17 +102,18 @@ def test_long_sni(self):
         """Also tests receiving of TLS alert."""
         self.start_all()
         hs12 = TlsHandshake()
-        hs12.sni = ["a" * 100 for i in range(10)]
+        hs12.sni = "a" * 1000
         # Tempesta must send a TLS alerts raising TLSProtocolError exception.
```

Comment:
Does this comment is still actual (here and in othere lines in test_hanshake_tls.py) ?

Reply:
Remove old comments